kubeflow/common/types_test.py

```
@@ -0,0 +1,109 @@
+
+        assert test_case.expected_status == SUCCESS, (
+            f"Expected exception but none was raised for {test_case.name}"
+        )
+
+        for key, expected_val in test_case.expected_output.items():
+            if key == "has_client_config":
+                assert isinstance(cfg.client_configuration, client.Configuration)
+            else:
+                assert getattr(cfg, key) == expected_val
+
+    except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)